hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/timeline/versioning/v2/LSMTimelineWriter.java

```diff
@@ -146,7 +147,7 @@ public void write(
           preWriteCallback.ifPresent(callback -> callback.accept(activeAction));
           // in local FS and HDFS, there could be empty completed instants due to crash.
           final HoodieLSMTimelineInstant metaEntry = MetadataConversionUtils.createLSMTimelineInstant(activeAction, metaClient);
-          writer.write(metaEntry.getInstantTime(), new HoodieAvroIndexedRecord(metaEntry), wrapperSchema);
+          writer.write(metaEntry.getInstantTime(), new HoodieAvroIndexedRecord(metaEntry), HoodieSchema.fromAvroSchema(wrapperSchema));
```

Comment:
Let's move this `fromAvroSchema` outside of the loop for now so we don't have so many conversions on this path.